src/components/CompletedTask/CompletedTask.test.tsx

```diff
@@ -0,0 +1,107 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+
+import CompletedTask from "./CompletedTask";
+
+it("should render the Completed Task component", () => {
+  render(
+    <CompletedTask
+      taskHeading={"Run 5k"}
+      category={"Fitness"}
+      points={10}
+      description={"Run as fast as you can"}
+      image={
+        "https://img.freepik.com/premium-photo/running-man-silhouette-sunset-time-sport-active-life-concept_221513-1606.jpg"
+      }
+    />
+  );
+  const taskHeading = screen.getByText("Run 5k");
+  const taskCategory = screen.getByText("Fitness");
+  const taskPoints = screen.getByText(/10/);
+  expect(taskHeading).toBeInTheDocument();
+  expect(taskCategory).toBeInTheDocument();
+  expect(taskPoints).toBeInTheDocument();
+});
+
+it("shouldn't display the collapsed information on initial render", () => {
+  render(
+    <CompletedTask
+      taskHeading={"Run 5k"}
+      category={"Fitness"}
+      points={10}
+      description={"Run as fast as you can"}
+      image={
+        "https://img.freepik.com/premium-photo/running-man-silhouette-sunset-time-sport-active-life-concept_221513-1606.jpg"
+      }
+    />
+  );
+  const taskMedia = screen.queryByText("Media");
+  const desc = screen.queryByText("Run as fast as you can");
+  const img = screen.queryByAltText("Run 5k");
+  expect(taskMedia).not.toBeInTheDocument();
+  expect(desc).not.toBeInTheDocument();
+  expect(img).not.toBeInTheDocument();
+});
+
+it("should display the collapsed information when the arrow icon is clicked", async () => {
+  render(
+    <CompletedTask
+      taskHeading={"Run 5k"}
+      category={"Fitness"}
+      points={10}
+      description={"Run as fast as you can"}
+      image={
+        "https://img.freepik.com/premium-photo/running-man-silhouette-sunset-time-sport-active-life-concept_221513-1606.jpg"
+      }
+    />
+  );
+  const arrowIcon = screen.getByTestId("ExpandMoreIcon");
+  fireEvent.click(arrowIcon);
+  expect(await screen.findByText("Media")).toBeInTheDocument();
+  expect(await screen.findByText("Run as fast as you can")).toBeInTheDocument();
+  expect(await screen.findByAltText("Run 5k")).toBeInTheDocument();
```

Comment:
Can you restructure these expect statements to use `queryByText` and store these queries in a variable as you've done in the other tests

Reply:
We have taken them out as seperate variables, but we are still using FindbyText. The clicking action has some animation associated with it, so we need to use await to wait for these elements to load before testing them.